src/queries/next-resource.sql

```diff
@@ -1,8 +1,9 @@
-SELECT req_url, count(req_http_X_CDN_Request_ID) AS visits, resp_http_Content_Type, status_code 
+SELECT req_url, count(req_http_X_CDN_Request_ID) AS visits, resp_http_Content_Type, status_code
+FROM ^tablename
```

Comment:
What does the `^` syntax do?

Reply:
Thank you @trieloff! The ^ syntax is for us to parameterize tablename etc! This is so that we don’t have to change the query file and redeploy the run-query service; we can just send a request with a table name injected!

Reply:
Very cool. Thanks for the explanation.

Reply:
You’re very welcome!